### ISSUE TITLE
Audio play error on IE 11

### ISSUE THREAD

**Issue by wzhonggo** [lhz516/react-h5-audio-player#98]
**Describe the bug**

```js
playAudioPromise = (): void => {
    this.audio.current
      .play()
      .then(null)
      .catch((err) => {
        const { onPlayError } = this.props
        onPlayError && onPlayError(new Error(err))
      })
  }
```

[Audio play return a Promise not support on IE](https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/play ), so playAudioPromise  method will fire error **Unable to get undefined or null referenced property "then"**.

**Environment**

Package version: 3.6.1
React version:  17.0.1
Browser and its version: IE 19041.804
OS and its version: Windows 10 Build 19041.804

**Comment by lhz516** [comment]
Ah, thanks for reporting. Working on it.

**Comment by lhz516** [comment]
Fixed in [v3.6.2](https://github.com/lhz516/react-h5-audio-player/releases/tag/v3.6.2)

**Comment by wzhonggo** [comment]
It is ok.
